src/lib/queries/get-exists.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { HomeDocument } from 'test/module'
+import { getExistsQuery } from './get-exists'
+
+describe('getExistsQuery', () => {
+    it('accepts only schema fields', () => {
```

Comment:
can we also add a test with `elasticsearch` so we can be more prepared for incoming `v8`? 🙏